addons/website/static/src/snippets/s_rating/options.js

```diff
@@ -123,18 +131,12 @@ options.registry.Rating = options.Class.extend({
      * @private
      */
     _renderIcons: function () {
-        const icons = {
-            'fa-star': 'fa-star-o',
-            'fa-thumbs-up': 'fa-thumbs-o-up',
-            'fa-circle': 'fa-circle-o',
-            'fa-square': 'fa-square-o',
-            'fa-heart': 'fa-heart-o'
-        };
-        const faClassActiveIcons = (this.iconType === "custom") ? this.faClassActiveCustomIcons : 'fa ' + this.iconType;
-        const faClassInactiveIcons = (this.iconType === "custom") ? this.faClassInactiveCustomIcons : 'fa ' + icons[this.iconType];
-        const $activeIcons = this.$target.find('.s_rating_active_icons > i');
-        const $inactiveIcons = this.$target.find('.s_rating_inactive_icons > i');
-        $activeIcons.removeClass().addClass(faClassActiveIcons);
-        $inactiveIcons.removeClass().addClass(faClassInactiveIcons);
+        const faClassIcons = (this.iconType === "custom") ? this.faClassActiveCustomIcons : 'fa ' + this.iconType;
+        const $ratingIcons = this.$target.find('.s_rating_icons i');
+        $ratingIcons.removeClass().addClass(faClassIcons);
+        this.$target[0].setAttribute("aria-label", _t("%(amount)s out of %(total)s stars", {
+            amount: this.nbActiveIcons || 0,
+            total: this.nbTotalIcons,
+        }));
```

Comment:
@chgo-odoo Ask for a review once you will have retested everything please... this is just broken:

- Set an active icon to some specific icon -> change the score of the rating -> all icons, including the non-active ones change to the "active" icon choice
- Set an inactive icon to some specific icon -> change the score of the rating -> you lose the icon choice
- ...

I don't think you have to change anything in this JS code besides removing the "active -> inactive icon map"...